homeassistant/components/ezviz/camera.py

```diff
@@ -192,7 +194,6 @@ def __init__(
         self._ffmpeg_arguments = ffmpeg_arguments
         self._ffmpeg = get_ffmpeg_manager(hass)
         self._attr_unique_id = serial
-        self._attr_name = self.data["name"]
```

Comment:
The cameras pull their name from the EZVIZ API. (As as per what the user configured in the EZVIZ app - example "Living room camera" or "Pet door")

Might be worth implementing in a @propery method so it updates if the user changes it?

Reply:
Doesn't update camera names without a reload of the integration. (If you were to change them).

There is a unique id issue that I'm planning to fix though. The older entities use their name in the unique_id which causes a mess if names are changed in the app.

Reply:
With this change, the camera entity gets the same name as the device (inherited from EzVizEntity). And that gets the name from the same place so I think after a reload this would still work.

I think renaming the device in the device registry when someone updates the name in the app would be an improvement for a future PR.

Reply:
Quick test from my side, work as expected.

Reply:
Do you agree with moving the whole device name from the API to a follow up PR?

Reply:
I agree. Was planning on refactoring the code in the camera entity as a last step.

Reply:
Good, can you approve this then? :)

Reply:
At least I have understood it like that.